src/main/java/com/example/springtweet/springtweet/controller/TwitterController.java

```diff
@@ -35,12 +35,25 @@
     }
 
     @PostMapping(value = "/tweet")
-    private ResponseEntity postTweet(@RequestParam(value = "file") MultipartFile file, @RequestParam String tweetMessage) throws CustomException, SuccessMessage {
+    public ResponseList<Status> postTweet(@RequestParam String tweetMessage) throws CustomException, SuccessMessage {
+        try {
+            Status status = twitter.updateStatus(tweetMessage);
+            String url = "https://twitter.com/" + status.getUser().getScreenName() + "/status/" + status.getId();
+            throw new SuccessMessage(url);
+        } catch (TwitterException e) {
+            e.printStackTrace();
+            throw new CustomException();
+        }
+    }
+
+    //post method for uploading image and tweet
+/*    @PostMapping(value = "/tweet")
+    public String postTweet(@RequestParam(value = "file") MultipartFile file, @RequestParam String tweetMessage) throws CustomException, SuccessMessage {
```

Comment:
You can have this method uncommented and have a different endpoint for it, say imagetweet, and just don't write a test case for that method since we couldn't resolve the final class mocking issue you were facing.